src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -85,87 +96,148 @@ When you run LDR in [`immediate` mode](#modes), you cannot replicate a table wit
     GRANT SYSTEM REPLICATION TO {your username};
     ~~~
 
-    If you need to change the password later, refer to [`ALTER USER`]({% link {{ page.version.version }}/alter-user.md %}).
+    To change the password later, refer to [`ALTER USER`]({% link {{ page.version.version }}/alter-user.md %}).
 
 ## Step 2. Connect from the destination to the source
 
-In this step, you'll set up an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) from the destination cluster to the source cluster. Depending on how you manage certificates, you must ensure that all nodes between the clusters have access to the certificate of the other cluster.
+In this step, you'll set up [external connection(s)]({% link {{ page.version.version }}/create-external-connection.md %}) to store the connection string for one or both clusters. Depending on how you manage certificates, you must ensure that all nodes between the clusters have access to the certificate of the other cluster.
 
 You can use the `cockroach encode-uri` command to generate a connection string containing a cluster's certificate.
 
-1. On the **source** cluster in a new terminal window, generate a connection string, by passing the replication user, node IP, and port, along with the directory to the source cluster's CA certificate:
+1. On the **source** cluster in a new terminal window, generate a connection string, by passing the user, node IP, and port, along with the directory to the source cluster's CA certificate:
```

Comment:
it would be good to clarify that the user in the external conn requires the REPLICATION priv.

Reply:
I've added this back in, not sure why I took that out while testing 🤔

Reply:
if you your use was `root`, then you had these privs automatically